### PR TITLE
So coords tweaks

### PR DESCRIPTION
**Pull request by pconesa**
This bring a pending internal refactorization of SetOfCoordinates I had for a long time plus other minor changes

### CODE REVIEW

**Review by JorMaister — APPROVED** [review]

File: tomo/protocols/protocol_import_tomograms.py [hunk 1]
```diff
@@ -26,25 +26,28 @@
...
 from pyworkflow.utils.path import createAbsLink, removeExt
 import pyworkflow.protocol.params as params
 
 from .protocol_base import ProtTomoImportFiles, ProtTomoImportAcquisition
-from ..objects import Tomogram
+from ..objects import Tomogram, SetOfTomograms
 from ..utils import _getUniqueFileName
 
 
+OUTPUT_NAME = 'Tomograms'
```
Comment (azazellochg) [inline tomo/protocols/protocol_import_tomograms.py:41]: Well, you just broke so many plugins by renaming the output of this import protocol...

Comment (fonsecareyna82) [reply tomo/protocols/protocol_import_tomograms.py:41]: Yes, we are fixing that in the rest of the plugins

**Comment by pconesa** [comment]
I'm aware most are fixed... already....should I go through yours?

**Comment by azazellochg** [comment]
True, but this constant did not exist before this PR. I fixed my Warp test, the rest are not my plugins

**Comment by pconesa** [comment]
Yes you are wrigt, probably for the next time we do this we should... first introduce the constant and adapt the plugins to us the constant .... then just change de constant value. Apologies for the mess.

We are trying to avoid outputs name like:

outputSetXXXXX

It is "noisy".

**Pull request merged**